app/ecs_run.py

```diff
@@ -0,0 +1,15 @@
+#!flask/bin/python
+import os
+import sys
+
+parent_dir = os.path.abspath(os.path.join(os.getcwd(), ".."))
+sys.path.append(parent_dir)
+sys.path.append(os.getcwd())
+
+os.environ["LOCATION"] = "LOCAL"
+
+from server import create_app
+
+app = create_app()
+
+app.run(host='0.0.0.0', port=9000)
```

Comment:
I think this will cause the app to be lunched using a werkzeug server, which isn't good for prod (https://github.com/pallets/flask/blob/master/src/flask/app.py#L768) - it should be using a uwsgi or similar prod server instead (as per the docker_app_script)